test/jasmine/tests/cartesian_interact_test.js

```diff
@@ -1449,7 +1449,7 @@ describe('axis zoom/pan and main plot zoom', function() {
             });
         });
 
-        it('panning a matching overlaying axis', function(done) {
+        it('@noCI panning a matching overlaying axis', function(done) {
```

Comment:
Instead of `@noCI` can we just increase the tolerance a bit?

Reply:
This test passes locally. We did reduce the number of `noCI` test when compared to pre-v2 era.
I hope in near future we could spend some time to bring that down to zero. 
This is I think possible by ensuring the needed fonts installed in the jasmine containers.
So if you don't mind I keep this flag and instead open an issue to track it?

Reply:
https://github.com/plotly/plotly.js/issues/5923

Reply:
Sure, that's fine. Thanks for the tracking issue.